src/CSS.php

```diff
@@ -66,6 +66,29 @@ public function setImportExtensions(array $extensions)
     }
 
     /**
+     * Move any import statements to the top
+     *
+     * @param $content string Nearly finished CSS content
+     *
+     * @return string
+     */
+    protected function moveImportsToTop($content) {
+
+        if (preg_match_all('#@import[^;]+;#', $content, $matches)) {
+
+            // remove from content
+            foreach($matches[0] as $import) {
+                $content = str_replace($import, '', $content);
```

Comment:
I don't think we should remove the @import occurences for specificity reasons.

Imagine this source code:

**a.css**
```css
body { color: red; }
```

**style.css**
```css
body { color: blue; }
@import url(a.css);
```

It would minify to:

**style.css**
```css
@import url(a.css);
body { color: blue; }
```

Because the import of a happens after style.css has already declared body color, the declaration on a.css would win (color would be red).
However, if we move the import to before the declaration, the last one defined would be blue (it would override red).

I think moving the assest imports up is a great idea. However, because of these rules with respect to specificity, I can't yet merge it as-is.
I think we can still move the imports to the beginning of the file, to make sure the browser starts loading them as soon as possible. However, we also need to keep them in place, to make sure the rules are not overwritten in another place. That would mean we have the imports in 2 places (at the top and in the original location), but the second one will probably just be read from cache anyway.

In short, my suggestion would be:

**style.css**
```css
@import url(a.css); /* This one to make sure browser starts loading quickly */
body { color: blue; }
@import url(a.css); /* This one to make sure it again overrides the above conflicting body color - it'll just be fetched from cache */
```


What do you think?